src/IO/H5/Python/CombineH5.py

```diff
@@ -0,0 +1,51 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+
+import click
+import rich
+
+import spectre.IO.H5 as spectre_h5
+
+
+@click.command(name="combine-h5")
+@click.option(
+    "--file-prefix",
+    required=True,
+    type=str,
+    help="prefix of the files to be combined (omit number and file extension)",
+)
+@click.option(
+    "--subfile-name",
+    "-d",
+    required=True,
+    type=str,
+    help="subfile name of the volume file in the H5 file (omit file extension)",
+)
+@click.option(
+    "--output",
+    "-o",
+    required=True,
+    type=click.Path(
+        exists=False,
+        file_okay=True,
+        dir_okay=False,
+        writable=True,
+        readable=True,
+    ),
+    help="combined output filename (omit file extension)",
+)
+
+def combine_h5_command(file_prefix, subfile_name, output):
+    """This executable is used for combining a series of HDF5 volume files into one
```

Comment:
As with the other CLI command you wrote, add a first line with a brief description, then an empty line, then details